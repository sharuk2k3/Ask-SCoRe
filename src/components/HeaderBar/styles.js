import {StyleSheet, Dimensions} from "react-native";

let deviceWidth = Dimensions.get("window").width;
let deviceHeight = Dimensions.get("window").height;

export default StyleSheet.create({
  HeaderStyle: {
    fontSize: 30,
    flex: 1,
    marginLeft: 10,
    marginTop: 5,
  },
  ImageView: {
    width: 55,
    height: 55,
    marginHorizontal: 5,
  },
  HeaderStyle1: {
    flexDirection: "row",
  },
  DividerView: {
    borderBottomColor: "black",
    borderBottomWidth: 1,
    marginBottom: 3,
    marginHorizontal: 10,
  },
});
