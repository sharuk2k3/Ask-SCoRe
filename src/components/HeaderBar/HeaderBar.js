import React from "react";
import {View, Text, Image} from "react-native";
import styles from "./styles";

export default class HeaderBar extends React.Component {
  render() {
    return (
      <View style={{marginBottom: 15}}>
        <View style={styles.HeaderStyle1}>
          <Image source={this.props.image} style={styles.ImageView} />
          <Text style={styles.HeaderStyle}>{this.props.title}</Text>
        </View>
        <View style={styles.DividerView} />
        <View style={styles.DividerView} />
      </View>
    );
  }
}
