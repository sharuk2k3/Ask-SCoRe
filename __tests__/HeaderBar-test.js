import React from "react";
import HeaderBar from "../src/components/HeaderBar/HeaderBar";
import renderer from "react-test-renderer";

jest.useFakeTimers();

test("renders Question Screen correctly", () => {
  const tree = renderer.create(<HeaderBar />).toJSON();
  expect(tree).toMatchSnapshot();
});
