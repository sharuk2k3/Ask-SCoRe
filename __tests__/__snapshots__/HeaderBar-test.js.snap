// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Question Screen correctly 1`] = `
<View
  style={
    Object {
      "marginBottom": 15,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Image
      style={
        Object {
          "height": 55,
          "marginHorizontal": 5,
          "width": 55,
        }
      }
    />
    <Text
      style={
        Object {
          "flex": 1,
          "fontSize": 30,
          "marginLeft": 10,
          "marginTop": 5,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "black",
        "borderBottomWidth": 1,
        "marginBottom": 3,
        "marginHorizontal": 10,
      }
    }
  />
  <View
    style={
      Object {
        "borderBottomColor": "black",
        "borderBottomWidth": 1,
        "marginBottom": 3,
        "marginHorizontal": 10,
      }
    }
  />
</View>
`;
